### PR TITLE
step 4, conclusion, and about the authors

### PR DESCRIPTION
**Pull request by 4383**

### CODE REVIEW

File: _drafts/contribute-to-ansible.markdown [hunk 1]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
     b. understand ansibullbot labels
 
 ## step 4 - What's happens next?
-    a. How to integrate modifications to the PR
-    b. obtain 2 shipit
 
-    Git
-    Github
+Now you have submitted your pull request to the official ansible project,
+you may be confronted with two possible cases.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: with -> to

File: _drafts/contribute-to-ansible.markdown [hunk 2]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+- First, after a code review maintainers can ask to you to apply changes in your code.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: can ask you

File: _drafts/contribute-to-ansible.markdown [hunk 3]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+- Second, your changes seems to looks good and you obtain ship it.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: Secondly, your changes looks good and  you gets two shipit.

File: _drafts/contribute-to-ansible.markdown [hunk 4]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+Now we want to explain how to deals with these use cases.
+
+#### a. How to integrate modifications to the PR
+
+After a code review the maintainers can ask to you to apply somes changes in your
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: - remove "the"  in "the mainteners"
- can ask you

File: _drafts/contribute-to-ansible.markdown [hunk 5]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+developments.
+
+If you want that your work to be integrated you have to apply these review comments, it's a sine qua non condition.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: If you wants your code being merged, you ...

File: _drafts/contribute-to-ansible.markdown [hunk 6]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+So in your local environment apply these changes on the branch corresponding to your pull request.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: the branch related to

File: _drafts/contribute-to-ansible.markdown [hunk 7]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+When you have fix the review comments, you can push your modifications into your fork:
+
+```sh
+$ git push origin <your-branch-name>
+```
+
+Then, your changes was automaticaly integrated to the already existing pull request. You have just to notify
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: are automatically integrated

File: _drafts/contribute-to-ansible.markdown [hunk 8]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+the comments author to review your changes.
+
+```
+@reviewer_name changes applied!
+```
+
+If your changes are accepted the reviewer can approval these by sending a shipit to your pull request.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: can approuve

File: _drafts/contribute-to-ansible.markdown [hunk 9]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+#### b. Obtain 2 shipit
+
+For your changes to be part of the official project you need to obtains at least 2 `shipit`.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: To get your changes merged in the official project ....

File: _drafts/contribute-to-ansible.markdown [hunk 10]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+Shipit is a keyword recognized by the ansibulbot when it was sending.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: Shipit is a keyword recognized by the ansibulbot. (does t need more I think)

File: _drafts/contribute-to-ansible.markdown [hunk 11]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+
+If your modifications are on a community module the ansibulbot can automaticaly merge your changes
+with the official repository source code, else, if your changes are on a core module,
+your modifications are merged by a core maintainer and can be rejected by him if something went wrong.
+
+The community is very active, code review and shipit can happen faster than you think.
+
+We recommand to you to check periodicaly the status of your merge request.
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: We recommands you to

File: _drafts/contribute-to-ansible.markdown [hunk 12]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+If your pull request doesn't move forward you can ask module maintainers for a review.
+
+Do not stay inactive and be proactive about your work!
 
 ## Conclusion
+
+Ansible's an indispensable tool, used by DevOps, system engineers, and a lot of
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: Ansible is an ...

File: _drafts/contribute-to-ansible.markdown [hunk 13]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+teams and projects all around the world. Have deep knownledge on this technology
+is a great assets to your career, specialy for DevOps/system jobs.
+In addition to that contributing to ansible help this product to be more efficient
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: In addition, contributing to ansible helpS

File: _drafts/contribute-to-ansible.markdown [hunk 14]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+and great!
+
+With this article we have introduce to you on how to become contributors and make
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: we have introduced

File: _drafts/contribute-to-ansible.markdown [hunk 15]
```diff
@@ -222,10 +222,82 @@ $ pip install -r ./test/runner/requirements/sanity.txt
...
+teams and projects all around the world. Have deep knownledge on this technology
+is a great assets to your career, specialy for DevOps/system jobs.
+In addition to that contributing to ansible help this product to be more efficient
+and great!
+
+With this article we have introduce to you on how to become contributors and make 
+ansible more understandable.
+
+We hope you liked our article,
+do not hesitate to post your comments and suggestions,
```
Comment (dj4ngo) [inline _drafts/contribute-to-ansible.markdown]: Do we have a comments module ? ;)

**Comment by 4383** [comment]
@dj4ngo review comments applied!

**Pull request merged**